services/email_verification.js

```diff
@@ -0,0 +1,78 @@
+/**
+* @file email_verification.js
+* @author Capuccino
+* @license BSD-3-Clause
+*/
+
+const micro = require("micro");
+const {json} = micro;
+const redis = require("redis");
+const Redite = require("redite");
+const config = require("./email.config");
+const snowflake = require("snowflake-codon");
+const client = redis.createClient({
+    host: config.redis.host,
+    password: config.redis.password
+});
+const db = new Redite({client});
+const idGen = new snowflake(12, Math.floor(Math.random() * 1129), 2018, 200);
+
+const mailer = require("nodemailer").createTransport({
+    host: config.host,
+    port: 465,
+    secure: true,
+    auth: {
+        user: config.auth.user,
+        pass: config.auth.password
+    }
+}, {
+    // sender options
+    from: 'Sayonika <noreply@sayonika.moe>',
```

Comment:
Does nodemailer ensure the email is verified? If not, it might end up in the `spam` directory, which we'd like to prevent.

Reply:
We use Mailgun so this shouldn't be a concern